Source/SwiftLintBuiltInRules/Rules/Idiomatic/NoMagicNumbersRule.swift

```diff
@@ -196,4 +203,24 @@ private extension ExprSyntaxProtocol {
         }
         return nil
     }
+
+    func isOperandOfBitwiseShiftOperation() -> Bool {
+        guard let siblings = parent?.as(ExprListSyntax.self)?.children(viewMode: .sourceAccurate) else {
+            return false
+        }
+        if siblings.count == 3 {
+            let index = siblings.index(after: siblings.startIndex)
+            if let tokenKind = siblings[index].as(BinaryOperatorExprSyntax.self)?.operatorToken.tokenKind {
+                if tokenKind == .binaryOperator("<<") || tokenKind == .binaryOperator(">>") {
+                    let lastIndex = siblings.index(after: index)
+                    let selfAsSyntax = self.as(Syntax.self)
+                    if siblings[siblings.startIndex] == selfAsSyntax || siblings[lastIndex] == selfAsSyntax {
```

Comment:
I cannot think of an example in which this condition would not be true. Omit it?